SampleApp/SampleApp/ViewController.swift

```diff
@@ -31,13 +31,7 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
-        var existingDeviceRiskUUID:String? = nil
-        
-        if let uuid = UserDefaults.standard.string(forKey: "DeviceRiskUUID") {
-            existingDeviceRiskUUID = uuid
-        }
-        
-        deviceRiskManager.setTracker(key: "Socure-public-key", sources:  [.device, .network, .accessibility, .locale, .advertising, .accelerometer,.magnetometer,.motion, .pedometer, .location], existingUUID: existingDeviceRiskUUID)
+        deviceRiskManager.setTracker(key: "Socure-public-key", sources:  [.device, .network, .accessibility, .locale, .advertising, .accelerometer,.magnetometer,.motion, .pedometer, .location], existingUUID: nil)
```

Comment:
We can remove the existingUUID argument altogether right? Let's just not pass it at all and let the **default** nil value be used

Reply:
done.